rlclientlib/live_model_impl.h

```diff
@@ -73,6 +74,7 @@ namespace reinforcement_learning
     error_callback_fn _error_cb;
     model_management::data_callback_fn _data_cb;
     utility::watchdog _watchdog;
+    decision_mode _decision_mode;
```

Comment:
1) Can we make this a bool like _bg_model_proc(essing)?
Making this an enum implies there are many behaviors.

2) The name decision_mode seems non specific.  Some thing more specific to describe the behavior would be better.  Will review the rest of the code to see if I have a better suggestion.

Reply:
From what I heard from Ed, more modes may come (e.g. logging mode). So a boolean may not be suffient for the design.

Reply:
should we call this a bool flag "logging_only"  because that's what is happening ... no?  The config file generation can set this as a fine grained behavior desired from rlclientlib.  Then we can mix and match fine grained behaviors to achieve different modes.

---
In reply to: [386531155](https://github.com/VowpalWabbit/reinforcement_learning/pull/129#discussion_r386531155) [](ancestors = 386531155)